src/js/angular/import/templates/exportSettingsModal.html

```diff
@@ -0,0 +1,57 @@
+<div class="modal-header">
+    <button type="button" class="close" ng-click="cancel()"></button>
+    <h4 class="modal-title">{{'export.settings' | translate}}</h4>
+</div>
+
+<div class="modal-body">
+    <form name="exportSettingsForm" id="exportSettingsForm" class="export-settings-form">
+        <div class="form-horizontal">
+            <div class="form-group row">
+                <label class="col-lg-3 col-form-label" for="wb-JSONLD-mode">
+                    {{'jsonld.mode' | translate}}
+                    <span class="btn btn-link p-0"
+                          uib-popover="{{'jsonld.mode.info' | translate}}"
+                          popover-trigger="mouseenter"
+                          popover-placement="right"><em class="icon icon-info"></em></span>
+                </label>
+                <div class="col-lg-9">
+                    <select class="form-control" id="wb-JSONLD-mode"
+                            ng-model="currentMode"
+                            ng-options="mode.link for mode in JSONLDModes">
+                    </select>
+                </div>
+            </div>
+
+            <div class="form-group row" ng-if="currentMode.name == 'framed' || currentMode.name == 'frame'">
```

Comment:
You have already defined the JSONLDModesNames model which should be the source of truth and yet here you hardcode these types. Why? And something more. comparison in javascript must be done always by `===` or `!==`. There should be a really special case in order to use `==`.
So you can extract the names in a enum like object like this
``` javascript
$scope.JSONLDModesNames = $scope.JSONLDModes.reduce(function (acc, cur) {
   acc[cur.name] = cur.name;
   return acc;
}, {});
```
and then check against it here
```
 ng-if="currentMode.name === JSONLDModesNames.framed || currentMode.name == JSONLDModesNames.frame"
```
It'd be even better if we can omit these ^ checks here by having two lists with the repective types like 
```
$scope.JSONLDFramedModes = [JSONLDModesNames.framed, JSONLDModesNames.frame];
$scope.JSONLDContextModes = [JSONLDModesNames.context, JSONLDModesNames.compacted, JSONLDModesNames.flattened];
```
and then here you can just do
```
ng-if="JSONLDFramedModes.indexOf(currentMode.name) > -1"
```